man/plot_layout.Rd

```diff
@@ -35,7 +35,8 @@ Returns a plotly object which contains the layout
   that is used for probability plotting.
 }
 \description{
-This function is used to create the layout of a probability plot.
+This function is used to create the layout of a probability plot. It is called
+by \code{\link{plot_prob}} internally.
```

Comment:
The function is called inside plot_prob to determine appearance of the grid with respect to the given x
... I think some more information would be nice at this point